# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../imagemagick/multilib-fix.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


On 64bit multilib-systems this had caused linking to 32bit libs.
	-- jsaw
re-diffed for 6.2.8 version - Aldas (Baldzius)

diff -Nur ImageMagick-6.2.8-orig/configure.ac ImageMagick-6.2.8/configure.ac
--- ImageMagick-6.2.8-orig/configure.ac	2006-07-03 15:29:56.000000000 +0000
+++ ImageMagick-6.2.8/configure.ac	2006-07-23 23:35:24.000000000 +0000
@@ -1926,7 +1926,7 @@
     then
       CPPFLAGS="$CPPFLAGS -I${xml2_prefix}/include/libxml2"
     fi
-    LDFLAGS="$LDFLAGS -L${xml2_prefix}/lib"
+    LDFLAGS="$LDFLAGS"
    fi
   failed=0;
   passed=0;
